.doc_gen/metadata/cognito-identity-provide_metadata.yaml

```diff
@@ -0,0 +1,17 @@
+# zexi 0.4.0
```

Comment:
Fix the typo in the filename: `cognito-identity-provider_metadata.yaml`.

Reply:
Fixed